claimstore/modules/claims/restful.py

```diff
@@ -446,3 +527,6 @@ def get(self):
 claims_api.add_resource(PredicateResource,
                         '/predicates',
                         endpoint='predicates')
+claims_api.add_resource(EquivalentIdResource,
+                        '/eqids',
+                        endpoint='eqids')
```

Comment:
The output of `/eqids` looks like:

```
[   
    {   
        "eqid": "80b1d6e1-3e8b-4b4f-8259-6e4709fa96d9",
        "id": 1,
        "type": "ARXIV_ID",
        "value": "cond-mat/9906097"
    },
    {   
        "eqid": "80b1d6e1-3e8b-4b4f-8259-6e4709fa96d9",
        "id": 2,
        "type": "DOI",
        "value": "C10.1103/PhysRevE.62.7422"
    },
    {   
        "eqid": "6a82e3a9-91b4-4e49-95ff-2d7344bd001d",
        "id": 3,
        "type": "INSPIRE_RECORD_ID",
        "value": "cond-mat/9906097"
    },
    {   
        "eqid": "6a82e3a9-91b4-4e49-95ff-2d7344bd001d",
        "id": 4,
        "type": "CDS_RECORD_ID",
        "value": "2003192"
    },
    {   
        "eqid": "6a82e3a9-91b4-4e49-95ff-2d7344bd001d",
        "id": 5,
        "type": "CDS_REPORT_NUMBER",
        "value": "CMS-PAS-HIG-14-008"
    }
]
```

This is OK, but if we want people to consume this output programmatically later, what about grouping by `eqid`? Kind of like:

```
{   
    "80b1d6e1-3e8b-4b4f-8259-6e4709fa96d9": [
       {
        "type": "ARXIV_ID",
        "value": "cond-mat/9906097"
       },
       {   
        "type": "DOI",
        "value": "C10.1103/PhysRevE.62.7422"
       }
     ],
     ...
}
```